linux/kernel/building.md

```diff
@@ -208,6 +251,17 @@ sudo umount mnt/fat32
 sudo umount mnt/ext4
 ```
 
+#### For 64-Bit
+
+```bash
+sudo cp mnt/fat32/$KERNEL.img mnt/fat32/$KERNEL-backup.img
+sudo cp arch/arm64/boot/Image mnt/fat32/$KERNEL.img
+sudo cp arch/arm64/boot/dts/broadcom/*.dtb mnt/fat32/
+sudo cp arch/arm64/boot/dts/overlays/*.dtb* mnt/fat32/overlays/
+sudo umount mnt/fat32
```

Comment:
This fails to copy the README.

Reply:
The readme is not generated on a his build so there is no file to copy

Reply:
What do you mean not generated in this build? If you are only updating the 64-bit kernel then the README should be copied.

Reply:
When I build the kernel with the 64bit instructions, there is no README file to copy like the 32bit kernel build. I cant add to the instruction to copy a file that does not exist..

Reply:
The overlays are built _from arch/arm_, and the README must be copied _from arch/arm_. Look at the 32-bit instructions and you might understand.

Reply:
interesting, i will do another build and check if it exists. If so, i will add it to the build instructions

Reply:
It's a source file - it's just copied.

Reply:
You are right, i was normally building into a separate folder so i got confused.

Fixed in f9e1697